common/include/pcl/experimental/executor/default/cuda_executor.hpp

```diff
@@ -0,0 +1,94 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2014-, Open Perception, Inc.
+ *  Author: Shrijit Singh <shrijitsingh99@gmail.com>
+ *
+ */
+
+#pragma once
+
+#ifdef CUDA
```

Comment:
`__CUDACC__` and other occurrences in this file.

Reply:
Will be resolved on v0.3 is integrated

Reply:
Done